packages/webpack/src/plugin.js

```diff
@@ -483,6 +483,8 @@ class LavaMoatPlugin {
         // I stole the idea from Zach of module federation fame
 
         const onceForChunkSet = new WeakSet()
+
+        options?.scuttleGlobalThis?.exceptions?.forEach((n, i, a) => a[i] = n.toString())
```

Comment:
- The callback to `forEach` returns `void`, so we should not return anything.
- Would prefer better variable names

```suggestion
        options?.scuttleGlobalThis?.exceptions?.forEach((exception, i, exceptions) => {
          exceptions[i] = exception.toString()
        })
```

or even 

```suggestion
        options?.scuttleGlobalThis?.exceptions?.forEach((exception, i, exceptions) => {
          exceptions[i] = `${exception}`
        })
```

`ScuttlerObjectConfig` needs an `exceptions` prop added to its definition.  This should fix the other type errors.

Reply:
Providing empty default values for these options may help avoid null-checking everywhere, too.

Reply:
i don't get ts, but this works :shrug: 74acc1ca62e0a5792faf137c7623b90ff8c41e8d